browser/js/products/product.factory.js

```diff
@@ -0,0 +1,14 @@
+app.factory('Product', function ($http) {
```

Comment:
Can you rename your factories "ProductFactory" so we can quickly identify it is a factory?